packages/itwinui-react/src/core/FileUpload/FileUploadTemplate.tsx

```diff
@@ -51,11 +68,52 @@ export const FileUploadTemplate = (props: FileUploadTemplateProps) => {
     acceptType,
     label = 'Choose a file',
     subtitle = 'or drag & drop it here.',
+    files,
+    fileLabel,
+    fileDescription,
+    svgIcon,
     children,
   } = props;
   useTheme();
 
-  return (
+  const icon = svgIcon ? (
```

Comment:
I think these icons could be mapped to file type (pdf for pdf, ppt to ppt icon, etc).

Reply:
it was my suggestion to not handle file extensions because we cannot handle every extension and it would bloat up the component. so it should be left upto the user.

but we should show examples of how the user would achieve it.